comp/etw/impl/etwSession.go

```diff
@@ -151,6 +151,35 @@ func (e *etwSession) StopTracing() error {
 	return globalError
 }
 
+func (e *etwSession) GetSessionStatistics() (etw.SessionStatistics, error) {
+	var stats etw.SessionStatistics
+	// it is not clear if we can safely reuse the properties buffer here
+	// so we allocate a new one
+	tmpBuf := make([]byte, len(e.propertiesBuf))
```

Comment:
are you copying it to simplify initializing it? Maybe we could create a a function that returns an initialized `EVENT_TRACE_PROPERTIES` buffer instead, to avoid potential issues with reusing one that's already filled out.

For instance, is this bit from the [docs](https://learn.microsoft.com/en-us/windows/win32/api/evntrace/nf-evntrace-controltracew) relevant here?
```
If you are reusing a EVENT_TRACE_PROPERTIES structure (i.e. using a structure that you previously passed to [StartTrace](https://learn.microsoft.com/en-us/windows/win32/api/evntrace/nf-evntrace-starttracew) or ControlTrace), be sure to set the LogFileNameOffset member to 0 unless you are changing the log file name, and be sure to set [EVENT_TRACE_PROPERTIES.Wnode.Flags](https://learn.microsoft.com/en-us/windows/win32/api/evntrace/ns-evntrace-event_trace_properties) to WNODE_FLAG_TRACED_GUID.
```

Reply:
I made a copy because the underlying `propertiesBuf` is used to stop the trace, and it wasn't clear to me if I would be polluting something that would be used later, so it seemed safer.  Since this intended to be used polling on some large (cpu wise) interval, the cost of doing the copy seemed worth the extra work.

But the root answer is that it wasn't clear from the existing implementation or the documentation what the effect of polluting this would be, so I went for safer.